apps/financial-aid/web-veita/src/components/MultiSelection/MultiSelection.css.ts

```diff
@@ -0,0 +1,12 @@
+import { style } from '@vanilla-extract/css'
+import { theme } from '@island.is/island-ui/theme'
+
+export const selectContainer = style({
```

Comment:
Is this necessary?  If we are using these styles for a Box component then we can just send it in as a prop:
display="block"